src/utils/helpers/validation/revalidateAll.ts

```diff
@@ -14,6 +14,20 @@ interface RevalidateAllProps {
   sectionCode?: string
   strict?: boolean
 }
+
+/**
+ * @function revalidateAll
+ * Run validation on each response (for question elements) in application
+ * using promises and some validation method - or the defaultValidation
+ * that uses the evaluationExpression for dynamic evaluations.
```

Comment:
I think it's *only* using the default validation method. Any custom validation is done within the plugin itself and just returned to the wrapper. Which means any custom validation is *not* being re-run whenever these revalidate functions are called. I don't think this is a problem right now, but we should probably bear it in mind.

Reply:
Hmmm good point. I'll update this comment. Although it might be fine, since the custom validation will trigger a change on the response which sends the mutation with `isValid: false` and when `useRevalidateApplication` runs each validation of responses is accounted as part of each section progress. I think this is ok, but need to test on one of our examples using the custom validation.

Reply:
Yeah, the thing is, with the way we've done it now, we don't have a way to force Custom validations to re-run as part of the whole "Revalidate" function. The plugin's own validations only run when they are rendered (or lose focus) and only then do they update their `isValid` value. 

Currently the only plugin that has a custom validation is the Password field, and it's not a problem in that case as it shouldn't be re-validated anyway. I think it's unlikely we'd have a plugin with custom validation for which its critical to get up-to-date revalidation, but we might need to revisit this if we can think of one. @andreievg ?